### ISSUE TITLE
Chrono dependency requirement is too low

### ISSUE THREAD

**Issue by samueltardieu** [Drakulix/simplelog.rs#28]
When using the minimally requested version, chrono 0.4.0 is used which in turns requires num 0.1.0 which does not compile anymore:

```
   Compiling num v0.1.0
error: expected identifier, found keyword `mod`
  --> /home/sam/.cargo/registry/src/github.com-1ecc6299db9ec823/num-0.1.0/src/bigint.rs:66:16
   |
66 | use std::str::{mod, FromStr};
   |                ^^^ expected identifier, found keyword
```

Chrono dependency must be bumped to at least 0.4.1. You can test this using `cargo-edit` and the command `cargo update -Z minimal-versions`. I suggest adding this to the CI as well.

PR pending.

**Comment by Drakulix** [comment]
Thanks. I am a bit short on time, I would appreciate another PR adding this to CI.

**Comment by Drakulix** [comment]
Fix released with v0.5.3.
